### DISCUSSION TITLE
Troubleshooting Foundry Setup in VS Code - Commands Not Found

### DISCUSSION THREAD

**Discussion by DonkoTonev** [Cyfrin/foundry-full-course-cu#1140] [Q&A]
I hope this message finds you well. I recently set up Foundry on my VS Code environment following the instructions provided. However, I've encountered some issues with the commands. When I attempt to run forge --version, I get the following error:

`ZOE ERROR (from forge): zoeParseOptions: unknown option (--version)
ZOE library version 2013-02-16
`

Additionally, the commands cast, anvil, and chisel are not recognized, and I receive the respective "command not found" messages.

Has anyone else encountered similar problems? Any guidance or suggestions on how to troubleshoot and resolve these command-related issues would be greatly appreciated.

Thank you in advance for your assistance!

**Comment by 0xusmanf** [comment]
Please check this [thread](https://ethereum.stackexchange.com/questions/135581/im-working-with-foundry-forge-seems-to-be-installed-but-i-get-zoe-error-whenev), it might help.
